Backpack/src/main/res/layout/view_bpk_skeleton_body_text.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@id/bpk_skeleton_container"
```

Comment:
rather than adding the ids to the id file you should be able to use `@+id` here I believe

Reply:
that's nice.